aks/Dockerfile

```diff
@@ -8,6 +8,8 @@ LABEL com.github.actions.description="Creates a service and a deployment with th
 LABEL com.github.actions.icon="triange"
 LABEL com.github.actions.color="blue"
 
+ENV GITHUB_ACTION_NAME="Deploy a container image to an AKS cluster"
+
```

Comment:
Check if there is any max length for the `AZURE_HTTP_USER_AGENT` header if there is, does this fit in the length?

Reply:
agreed, we could have a key like "AKS_ACTION".
Will check with PM